clustermesh/clustermesh.go

```diff
@@ -1810,3 +1810,131 @@ func DisableWithHelm(ctx context.Context, k8sClient *k8s.Client, params Paramete
 	_, err = helm.Upgrade(ctx, k8sClient.RESTClientGetter, upgradeParams)
 	return err
 }
+
+// ConnectWithHelm enables clustermesh using a Helm Upgrade action Certificates are generated via
```

Comment:
nit: It seems like this docbloc had more punctuation or newlines that disappeared.